Sources/SwiftParser/Patterns.swift

```diff
@@ -217,7 +229,10 @@ extension Parser {
     switch self.at(anyIn: MatchingPatternStart.self) {
     case (.var, let handle)?,
       (.let, let handle)?,
-      (.inout, let handle)?:
+      (.inout, let handle)?,
+      (.mutating, let handle)?,
+      (.borrowing, let handle)?,
+      (.consuming, let handle)?:
```

Comment:
Same comment of using `ValueBindingPatternSyntax.BindingSpecifierOptions` also applies here.